Source/Core/VideoBackends/Metal/MTLObjectCache.mm

```diff
@@ -220,7 +220,7 @@ static MTLCullMode Convert(CullMode cull)
   switch (cull)
   {
   case CullMode::None:
-  case CullMode::All:  // Handled by disabling rasterization
+  case CullMode::All:  // Handled in CPU
     return MTLCullModeNone;
```

Comment:
Specifically, it's done here:

https://github.com/dolphin-emu/dolphin/blob/d04e1ca96a705f606d5ba75f3bb915a0ea84571a/Source/Core/VideoCommon/VertexLoaderManager.cpp#L358-L362

I'd write this as "Handled by VertexLoaderManager::RunVertices" instead of just "in CPU".

I'm not sure why that code doesn't skip lines and points; this might be a hardware quirk (since I don't think lines/points can be drawn in such a way that backface culling applies, and they might not be culled at all; they're not culled in the software renderer as well).

In any case, the only pipeline that _does_ implement `CullMode::All` on the GPU is Vulkan (although apparently `VK_CULL_MODE_FRONT_AND_BACK` it [only applies to triangles](https://books.google.com/books?id=XVBxDQAAQBAJ&pg=PT338&lpg=PT338&q=VK_CULL_MODE_FRONT_AND_BACK)).

<details><summary>Relevant backend code</summary>

https://github.com/dolphin-emu/dolphin/blob/d04e1ca96a705f606d5ba75f3bb915a0ea84571a/Source/Core/VideoBackends/OGL/OGLRender.cpp#L1125-L1135
https://github.com/dolphin-emu/dolphin/blob/d04e1ca96a705f606d5ba75f3bb915a0ea84571a/Source/Core/VideoBackends/D3D/D3DState.cpp#L441-L442
https://github.com/dolphin-emu/dolphin/blob/d04e1ca96a705f606d5ba75f3bb915a0ea84571a/Source/Core/VideoBackends/D3D12/DX12Pipeline.cpp#L64-L66
https://github.com/dolphin-emu/dolphin/blob/d04e1ca96a705f606d5ba75f3bb915a0ea84571a/Source/Core/VideoBackends/Vulkan/VKPipeline.cpp#L43-L45

</details>

Reply:
> I'd write this as "Handled by VertexLoaderManager::RunVertices" instead of just "in CPU".

Done

> I'm not sure why that code doesn't skip lines and points; this might be a hardware quirk (since I don't think lines/points can be drawn in such a way that backface culling applies, and they might not be culled at all; they're not culled in the software renderer as well).

Yeah lines/points will have their cull mode set to none before getting sent to the render backend (which is convenient for anyone who might be trying to expand them to triangles on the GPU)
https://github.com/dolphin-emu/dolphin/blob/d04e1ca96a705f606d5ba75f3bb915a0ea84571a/Source/Core/VideoCommon/RenderState.cpp#L16-L18